ping/ping.h

```diff
@@ -63,11 +63,11 @@
 #define	DEFDATALEN	(64 - 8)	/* default data length */
 
 #define	MAXWAIT		10		/* max seconds to wait for response */
-#define MININTERVAL	10		/* Minimal interpacket gap */
-#define MINUSERINTERVAL	2		/* Minimal allowed interval for non-root */
+#define MIN_INTERVAL	10		/* Minimal interpacket gap */
+#define MIN_USER_INTERVAL	2		/* Minimal allowed interval for non-root */
```

Comment:
If we are doing the rename we may as well add units to the name, e.g. `MIN_INTERVAL_MS` and `MIN_USER_INTERVAL_MS`